steps/genymotion-cloud-saas-stop/0.1.0/step.yml

```diff
@@ -0,0 +1,38 @@
+title: Genymotion Cloud Stop Android Devices
+summary: |
+  Stop Genymotion Cloud SaaS instances
+description: |
+  Stop Genymotion Cloud SaaS instances.
+website: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop
+source_code_url: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop
+support_url: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop/issues
+published_at: 2020-02-20T09:58:40.787674451+01:00
+source:
+  git: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop.git
+  commit: c08619fc173ecb1dc2466a11478356eb35d50513
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
+run_if: ""
+inputs:
+- instance_uuid: $GMCLOUD_SAAS_INSTANCE_UUID
+  opts:
+    description: Instance UUIDs list you want to stop.
```

Comment:
I suggest highlighting the expected format of the list input or the expected separator (`,`).

Reply:
Hi @godrei, Thanks for the review. I made changes , so  will close this PR and open a new one.